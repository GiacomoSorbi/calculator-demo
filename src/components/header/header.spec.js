import * as React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from './'

describe('Header should', () => {
  it('exist', () => {
    expect(Header).toBeDefined()
  })

  it('have a consistent structure', () => {
    const wrapper = mount(<Header />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
