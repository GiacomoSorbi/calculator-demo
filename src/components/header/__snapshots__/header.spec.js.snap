// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should have a consistent structure 1`] = `
<Header>
  <div
    className="app-header"
  >
    <img
      alt="equal experts"
      className="logo"
      src="logo.svg"
    />
  </div>
</Header>
`;
